src/main/app/claims/models/Direction.ts

```diff
@@ -0,0 +1,35 @@
+import { Moment } from 'moment'
+import * as _ from 'lodash'
+import { MomentFactory } from 'shared/momentFactory'
+
+export interface Direction {
```

Comment:
Is there a better name? Direction seems very generic. is this directionOrder? but i see some otherdirectionOrder, so don't know what this is...

Reply:
each directions order can have multiple directions that need to be complied with. Directions might be a better name as it does return the array of directions